iroh/src/client/docs.rs

```diff
@@ -378,14 +384,14 @@ impl Doc {
         }))
     }
 
-    /// Get status info for this document
+    /// Returns status info for this document
     pub async fn status(&self) -> anyhow::Result<OpenState> {
         self.ensure_open()?;
         let res = self.rpc(StatusRequest { doc_id: self.id() }).await??;
         Ok(res.status)
     }
 
-    /// Set the download policy for this document
+    /// Sets the download policy for this document
```

Comment:
Things like this should probably explain a bit what download policies are and what is available and how they affect things.

But I'll take any improvement, so like this is also fine.

Reply:
Agreed. Happy to pick this up on another docs day :)